backend/internal/models/user.go

```diff
@@ -13,3 +13,14 @@ type User struct {
 	CreatedAt      time.Time `json:"created_at"`
 	UpdatedAt      time.Time `json:"updated_at"`
 }
+
+type Notification struct {
+	ID               int       `json:"id"`
+	Thumbnail        string    `json:"thumbnail"` // Url of the notification thumbnail
```

Comment:
where do the thumbnails come from? like what are the possible values

Reply:
i'm thinking to leave it flexible for now bc im not sure how we wanna branch this out- but usually from either a user pfp (if its like a follow) or maybe album cover (if the notification is regaurding a review). Unsure though - thoughts?

Reply:
ok that makes sense! my only fear here would be a user changing their pfp and it not being updated or something - so maybe the URL should be populated on the fly from the getNotifications endpoint (like if it's type follow, we get the follower's current PFP, if it's something related to a track/album we get the cover, etc.)

not sure if it's worth the extra lookup time though but just my thoughts